src/static/script.js

```diff
@@ -870,7 +870,7 @@ Mode.addEventListener('click', function () {
             }
         });
 
-        if (firstAvailableOption) {
+        if (currentBrush=='eraser') {
```

Comment:
Please confirm if 'eraser' is valid. In the latest revisions the neutral/background class is called 'background' instead of 'eraser'.